### PR TITLE
[INT-2348] Sample config files for differential privacy

### PR DESCRIPTION
**Pull request by walston**
# Problem

NavFT will soon support differential-privacy while we have some sample configuration files for this, they are hidden away from use-cases.

Blueprints in Console is attempting to expose differential privacy configs and adding a new field is the most efficient way to solve for this.

### CODE REVIEW

**Comment by walston** [comment]
Waiting on #464

File: docs/notebooks/demo/navft-dp/navft-dp-experiments.ipynb [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+    "axs[0].set_ylim([55, 95])\n",
+    "axs[1].set_ylim([85, 95])\n",
+    "fig.set_tight_layout(tight=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Trends observed are as expected: SQS go down as we decrease epsilon, and DPS goes up. The percentage of valid records is 100% due to the use of structured generation. Training and generation time increase considerably, due to changes required for DP."
```
[the remark this answers is not in the document]
Comment (walston) [reply docs/notebooks/demo/navft-dp/navft-dp-experiments.ipynb:277]: Should've had you review the python PRs I was building off of 😆

**Review by maxdubrinsky — APPROVED** [review]

File: docs/notebooks/demo/navft-dp/navft-dp-experiments.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreatgretel) [reply docs/notebooks/demo/navft-dp/navft-dp-experiments.ipynb:277]: 😬

**Comment by walston** [comment]
@andreatgretel good catch. I just removed the commits I pulled in from your PR since they're merged into develop now.

**Pull request merged**